Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ruby:2.4.1
+
+LABEL maintainer Travis CI GmbH <support+travis-app-docker-images@travis-ci.com>
+
+RUN apt-get update && apt-get upgrade -y --no-install-recommends && apt-get install -y postgresql
+
+# throw errors if Gemfile has been modified since Gemfile.lock
+RUN bundle config --global frozen 1
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY Gemfile      /usr/src/app
+COPY Gemfile.lock /usr/src/app
+
+RUN bundle install --deployment
+
+COPY . /usr/src/app
+
+RUN apt-cache search postgres
+# Install sqitch so migrations work
+RUN env BINDIR=/usr/local/bin /usr/src/app/script/install-sqitch
```

Comment:
OH! Seems my latest file from yesterday (still local) fixes some of these pathing issues and getting new errors. 

```root@1685f6f38fbf:/usr/src/app# sqitch deploy
DBD::Pg 2.0 required to manage PostgreSQL```

I think I see what's happening though (and naturally it's the end of the day so I need to go). Looking at the travis.yml looks like I need the `postgresql-server-dev-9.6` or similar package? 

Also we only have 9.3 on Enterprise right now, does any of this require 9.6? After some of this docker stuff that's another migration we'd like to make.